Makefile

```
@@ -0,0 +1,30 @@
+.PHONY: check tidy deps \
+	lint lint-md lint-go \
+	lint-fix lint-fix-md lint-fix-go
+
+check:
+	go test ./...
+
+tidy:
+	go mod tidy
+
+deps:
+	go install github.com/mgechev/revive@latest
+	go install golang.org/x/tools/cmd/goimports@latest
+	npm install
+
+lint: lint-md lint-go
+lint-fix: lint-fix-md
```

missing `lint-fix-go`
```suggestion
lint-fix: lint-fix-md lint-fix-go
```